style.css

```diff
@@ -0,0 +1,695 @@
+  margin: 0;
+  padding: 0;
+  font-size: small;
+  max-width: 100%;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+a {
+  text-decoration: none;
+}
+
+nav a {
+  transition: ease 0.5s;
+}
+
+.headerTop a {
+  color: aliceblue;
+}
+
+.mainNavBar a {
+  color: grey;
+}
+
+.about a {
+  color: red;
+}
+
+.logoSite {
+  display: flex;
+  height: 10%;
+  margin-top: 10px;
+}
+
+.logoSite b {
+  padding-top: 10px;
+  color: #ec534298;
+  font-family: serif;
+  font-size: 20px;
+}
+
+.pgrm_btn2 a {
+  display: none;
+  color: transparent;
+  color: aliceblue;
+}
+
+.pgrm-footer a {
+  color: aliceblue;
+  padding-bottom: 1px;
+  font-size: 20px;
+}
+
+nav a:hover {
+  color: cadetblue;
+  border-bottom: cadetblue 4px solid;
+}
+
+.headerTop {
+  display: flex;
+  list-style: none;
+  gap: 20px;
+  justify-content: flex-end;
+  padding: 10px;
+  padding-right: 15rem;
+  background-color: rgb(78, 77, 77);
+}
+
+.logo {
+  height: 5rem;
+}
+
+.headerBtm {
+  display: flex;
+  width: 70%;
+  justify-content: space-between;
+  justify-items: center;
+  margin: 0 auto;
+  margin-top: -20px;
+}
+
+.mainNavBar {
+  display: flex;
+  list-style: none;
+  gap: 50px;
+  padding: 2rem;
+  padding-bottom: 0.5rem;
+}
+
+.compain {
+  margin-top: -15px;
+  color: red;
+  border: 5px solid red;
+  padding: 10px;
+}
+
+main {
+  background-image: url(./icon/main-bg.jpg);
+}
+
+main .main {
+  padding-left: 15rem;
+  padding-top: 10rem;
+  padding-bottom: 7rem;
+}
+
+.title {
+  font-size: 45px;
+  width: 50%;
+}
+
+.title h1 {
+  color: rgba(237, 100, 100, 0.616);
+  margin-top: -10px;
+  font-weight: 900;
+  background-image: url(./icon/seduction.jpg);
+  -webkit-background-clip: text;
+}
+
+.title i {
+  font-size: 28px;
+  color: #fc2f18;
+}
+
+.intro {
+  border: solid 2px rgb(255, 255, 255);
+  background-color: rgb(207, 201, 201);
+  width: 40rem;
+  margin-top: 20px;
+}
+
+.intro p {
+  font-size: 12px;
+  padding: 15px 10px;
+  line-height: 20px;
+}
+
+.timeZone {
+  margin-top: 20px;
+  color: #272a31;
+}
+
+.timeZone h1 {
+  font-size: 25px;
+  margin-bottom: 10px;
+}
+
+.timeZone .p3 {
+  font-size: 18px;
+}
+
+main .fa-solid {
+  display: none;
+}
+
+button {
+  border: none;
+  background: none;
+}
+
+.pgrm-section {
+  color: #d3d3d3;
+  background-image: url(./icon/next-bg.jpg);
+  height: 40rem;
+}
+
+img {
+  position: relative;
+  object-fit: cover;
+}
+
+.pgrm img {
+  width: 50px;
+  padding-top: 10px;
+  filter: invert(100%);
+  justify-content: center;
+  margin: 0 auto;
+  object-fit: scale-down;
+  height: 35px;
+}
+
+.main-pgrm {
+  padding-top: 30px;
+  width: 13rem;
+  justify-content: center;
+  margin: 0 auto;
+}
+
+.pgrm-title {
+  font-size: 25px;
+}
+
+.pgrm-border {
+  color: transparent;
+  margin-left: 33%;
+}
+
+.pgrm-option {
+  display: flex;
+  width: 60%;
+  justify-content: center;
+  margin: 0  auto;
+  margin-top: 50px;
+}
```

Comment:
- [ ] Great job, when I make the screen size 900px the text inside the card goes outside and isn't displayed properly please check the website is properly displayed for max and min, width on a desktop device.
- [ ] In the design there are spaces between individual cards please try to integrate that on your site.
- [ ] Please add the cover effect, which shows a white border when a user hovers over a card.


### Your Implementation
![image](https://user-images.githubusercontent.com/59416136/163625555-e6ae252b-5754-4914-92db-f4dc24765242.png)

### Behance Design
![image](https://user-images.githubusercontent.com/59416136/163625722-f2500e9c-256e-4183-ab28-c07481a3d1e6.png)